### PR TITLE
added fork me(github link)

### PR DESCRIPTION
**Pull request by aswinzz**
This PR is for the issue #293 

screenshot from 2017-02-17 18-15-18

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -188,10 +177,28 @@
         <span>Trailer</span>
       </a>
    </li> 
-  </ul>
+     <li>
```
Comment (gauthamzz) [inline _includes/navbar.html]: wrong indendation

File: _includes/navbar.html [hunk 2]
```diff
@@ -123,22 +123,11 @@
...
-    }(document, 'script', 'facebook-jssdk'));</script>
-    <div id="overlay">
-<div class="spinner"></div> 
-</div>
+
 <!-- Navbar-Start -->
 <!--Mobile Navigation Bar -->
 <div class="navbar-static-top visible-xs-block">
-<div id='snav' class='en'>
+   <div id='snav' class='en'>
```
Comment (gauthamzz) [inline _includes/navbar.html]: wrong indendation

**Comment by aswinzz** [comment]
now?

**Comment by gauthamzz** [comment]
LGTM

**Pull request merged**